app/views/comments/_comment_date.erb

```diff
@@ -1,7 +1,15 @@
 <div class="comment-date">
   <a href="<%= decorated_comment.path %>">
+  <span class="hidden m:inline-block">
```

Comment:
What's this line for? Won't it hide the time, always?

Reply:
Someone suggested it above. Seems it's to make the span take less space on mobile. You know, kinda make it stacked?

Reply:
OK I'll get clarification on that, I don't think they meant to have it around that section though but rather around a new bit not in your code yet.

So you can remove it for now.